Source/DafnyRuntime/DafnyRuntime.py

```diff
@@ -461,6 +470,7 @@ def __iter__(self):
 class defaults:
     bool = staticmethod(lambda: False)
     char = staticmethod(lambda: 'D')
+    codepoint = staticmethod(lambda: CodePoint('D'))
```

Comment:
```suggestion
    codepoint = staticmethod(lambda: CodePoint(char))
```
I _think_ that works.

Reply:
Not quite, but this does:

```suggestion
    codepoint = staticmethod(lambda: CodePoint(defaults.char()))
```